_posts/en/2020-05-20-Team-Topologies-Whitelisting-app.md

```diff
@@ -64,9 +66,9 @@ Both of the above two sections have been focused on increasing flow of business
 Is this a competency required for the duration of the project? If not, a new team member is not the optimal choice, as it is preferable to not interchange members of the team frequently.
 Do not grow the team too large. Ideally we want to keep the agile teams as small as possible, given that the required competencies to deliver business value exist within the team. That means, all things being equal, it is preferable to enhance the skills of a team rather than onboard a new person, if this is an option.
 
-If the team requires a skill that it is missing, this is where an enabling team comes it. It exists temporarily and collaborates with the team for a short time, in order to get the product team up to speed. During this time, the interaction method would be one of collaboration (this is in contrast with the interaction mode between teams, which is XaaS). If the team requires some functionality that requires very difficult skill sets than the ones existing within the team, this may be a natural fracture plan whereby one could create a separate team, and then have the teams communicate via XaaS rather than through direct collaboration in an enabling capacity. For further discussion regarding the details of this approach, it is recommended to read Team Topologies, referenced to and linked to above.
+If the team requires a skill that it is missing, this is where an enabling team comes it. It exists temporarily and collaborates with the team for a short time, in order to get the product team up to speed. During this time, the interaction method would be one of collaboration (this is in contrast with the interaction mode between teams in Figure 1, which is XaaS). If the team requires some functionality that requires very difficult skill sets than the ones existing within the team, this may be a natural fracture plane whereby one could create a separate team, and then have the teams communicate via XaaS rather than through direct collaboration in an enabling capacity. For further discussion regarding the details of this approach, it is recommended to read Team Topologies, referenced to and linked to above.
```

Comment:
first sentence..."this is where an enabling team comes IN' (caps for emphasis) :-)
in the second sentence we say 'It exists temporarily'...what is it? is this a human?
In the sentence "if the team requires functionality that requires very difficult skill sets'.  what is a difficult skill set?  do we mean different (from what exists within the team)?
What is a 'natural fracture plane'....will most readers know what this is?

Reply:
Fixed in "Changing phrasing to that recommnded in the comments"